src/components/Button/Button.tsx

```diff
@@ -10,15 +9,9 @@ const Button: React.FC<ButtonProps> = ({
   bgColor,
   color,
 }) => {
-  const classes = cn(styles.button, {
-    [styles.outline]: style === "outline",
-    [styles.text]: style === "text",
-  });
+  const classes = getButtonClasses({ style });
```

Comment:
I can see why you were excited by this the getButtonClasses function does make things look much neater.